func_adl_xAOD/cms/aod/event_collections.py

```diff
@@ -1,3 +1,5 @@
+import func_adl_xAOD.common.cpp_types as ctyp
+
```

Comment:
Run the command "Organize Imports" in vscode to sort and make pretty import statements.

Reply:
Which all files does this needs to be run in?

Reply:
> Which all files does this needs to be run in?
@gordonwatts Could you please answer this